### PR TITLE
Add birthday cake tradition to company events list

### PR DESCRIPTION
**Pull request by bzz**
Documenting a long-standing tradition of Madrid's office.

\cc @dennwc @ricardobaeta @eiso @kuba-- and @zurk

### CODE REVIEW

File: office/company_events.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ Some activities let us be more flexible (beers) while others have to follow a sp
 ## Friday Lunchs
 Once a month we organise a Friday lunch at the office. It is the fourth Friday of the month. Every sourcerer is invited to join. We order food (different every month) for everybody and we share good moments together. 
 
+## Birthday cakes
+Shared calendar `KinHR` includes birthdays of all sourcerers. Every time there is an event in that calendar - a cake is going to be shared in the office on that day, around 17:00. If the birthday is on weeked, the cake will happen on Monday. 
+
+Everyone in the company, including people who work remote, is very wellcome to join either at the kitchen or at https://appear.in/sourced-birthday
```
Comment (tsolakoua) [inline office/company_events.md]: typo: wellcome

**Comment by bzz** [comment]
Thank you for the prompt feedback, @tsolakoua ! Addressed in f5fa328

**Comment by estherrgarcia** [comment]
I'm sorry for remote people but I don't see the sense of sending a cake (i don't want to imagine the work to find a place in every country to send a cake to remote's houses). We are doing at the office because it is a nice way to share time together and be nice with our colleagues. It's the same for a get-together or when we go out to have lunch together a random day. It's something that we do because we are in the same place. 
For me, it would be more important if my colleagues say happy birthday to me than the company sending a cake to me.

**Comment by bzz** [comment]
> company sending a cake to me

I hope that the text in this PR does not imply such suggestion. If it does, sorry, it was not intended.

Being remote is longly enough so just having a chance to see everyone at the same place having a 🍰 is already rewarding. 

As a remote first company, and from my experience, I think it's worth to have those occasions pointed out in the guide so people do not feel excluded.

@eiso Will update the text to make it clear that it's about the tradition at the physical office in Madrid.

**Comment by bzz** [comment]
@eiso feedback addressed in 7c8402b

**Comment by bzz** [comment]
@tsolakoua feedback addressed in ea0b617

**Pull request merged**